pkg/csi/service/wcpguest/controller.go

```diff
@@ -716,6 +717,10 @@ func controllerUnpublishForBlockVolume(ctx context.Context, req *csi.ControllerU
 	}
 	var err error
 	if err := c.vmOperatorClient.Get(ctx, vmKey, virtualMachine); err != nil {
+		if apierrors.IsNotFound(err) {
```

Comment:
Can we add a check if err is not nil before calling `apierrors.IsNotFound(err)`?